src/index.js

```diff
@@ -101,26 +126,40 @@ AudioEngine.prototype.waitForBeats = function (beats) {
     });
 };
 
+/**
+ * Set the global tempo in bpm (beats per minute)
+ * @param {number} value - the new tempo to set
+ */
 AudioEngine.prototype.setTempo = function (value) {
     this.currentTempo = value;
 };
 
+/**
+ * Change the tempo by some number of bpm (beats per minute)
+ * @param  {number} value - the number of bpm to change the tempo by
+ */
 AudioEngine.prototype.changeTempo = function (value) {
     this.setTempo(this.currentTempo  + value);
 };
 
+/**
+ * Create an AudioPlayer. Each sprite or clone has an AudioPlayer.
+ * It includes a reference to the AudioEngine so it can use global
+ * functionality such as playing notes.
+ * @return {AudioPlayer}
+ */
 AudioEngine.prototype.createPlayer = function () {
     return new AudioPlayer(this);
 };
 
-/* Audio Player
-
-Each sprite or clone has an audio player
-the audio player handles sound playback, volume, and the sprite-specific audio effects:
-pitch and pan
-
-*/
 
+/**
+ * Each sprite or clone has an audio player
+ * the audio player handles sound playback, volume, and the sprite-specific audio effects:
```

Comment:
Looks like some messy editing here :)

I noticed another few cases where the line length was kind of weird, with a small line above an 80(ish)-char line..